bridges/ReutersBridge.php

```diff
@@ -202,6 +205,22 @@ private function handleArticleContent($contents) {
 				case 'p_table':
 					$description = $description . $content['content'];
 					break;
+				case 'upstream_embed':
+					$media_type = $content['media_type'];
+					$cid = $content['cid'];
+					$embed = '';
+					if($media_type == 'tweet') {
+						$embed .= "<iframe 
+													src=\"https://platform.twitter.com/embed/Tweet.html?id=$cid\" 
+													title=\"Twitter Tweet\"
+													scrolling=\"no\" 
+													frameborder=\"0\" 
+													allowtransparency=\"true\" 
+													allowfullscreen=\"true\" 
+													style=\"width: 550px;height: 225px;\">
+											</iframe>";
```

Comment:
Why so much indent?

Reply:
I try to indent like this in VS Code
![image](https://user-images.githubusercontent.com/33269526/121107057-6f3aa880-c831-11eb-89f4-56b982c0d92f.png)

but it turn out like that, weird.

Reply:
Indent issue is fixed in new commit.